lua/efmls-configs/formatters/deno-fmt.lua

```diff
@@ -0,0 +1,15 @@
+-- Metadata
+-- languages: javascript,typescript
```

Comment:
I see that it can format other files too, it'd be nice to add them here as well like `javascriptreact` (jsx), `typescriptreaact` (tsx), etc. for documentation.

Taking from https://docs.deno.com/runtime/manual/tools/formatter

Reply:
sure and I just added! looks like this is the first one to declare for `javascriptreact` and `typescriptreact` in the metadata.

However, unrelated to this PR and just a question, it seems to be the metadata is only being used for documentation and users still manually need to do something like below, correct?

```lua
local denofmt = require 'efmls-configs.formatters.deno-fmt'
local stylua = require 'efmls-configs.formatters.stylua'

local languages = {
  javascript = { denofmt },
  javascript = { denofmt },
  javasjavascript = { denofmt },
  typesjavascript = { denofmt },
  lua = { stylua },
}

-- user's configuration continues ...
```

I guess the manual way has pros and cons:
- cons: language to tool is not matched out of the box
- pro: you can easily opt out for a language that you don't wan t a tool to work for

In my case, pros outweighs the cons but not sure if _the metadata being used is only for documentation_ is clearly communicated (in case if not, that could be improved).

Reply:
> However, unrelated to this PR and just a question, it seems to be the metadata is only being used for documentation and users still manually need to do something like below, correct?

Yes, that's correct it's only used for documentation where they can refer to check what linters/formatters they can use for a language. I'm by no means forcing the language to a tool (except for when they want to use the defaults), this just creates the table that efm-langserver wants.